vector.py

```diff
@@ -0,0 +1,113 @@
+from random import randint, random
+import math
+
+class Vector(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    # str(a)
+    def __str__(self):
+        return "[" + str(self.x) + ", " + str(self.y) + "]"
+
+    # repr(a) or print(a)
+    def __repr__(self):
+        return self.__str__()
+
+    # add(b)
+    def add(self, b):
+        return Vector(self.x + b.x, self.y + b.y)
+
+    # a + b
+    def __add__(self, b):
+        return self.add(b)
+
+    # addAccumulate(b)
+    def addAccumulate(self, b):
+        self.x += b.x
+        self.y += b.y
+        return self
+
+    # a += b
+    def __iadd__(self, b):
+        return self.addAccumulate(b)
+
+    # sub(b)
+    def sub(self, b):
+        return Vector(self.x - b.x, self.y - b.y)
+
+    # a - b
+    def __sub__(self, b):
+        return self.sub(b)
+
+    # subAccumulate(b)
+    def subAccumulate(self, b):
+        self.x -= b.x
+        self.y -= b.y
+        return self
+
+    # a -= b
+    def __isub__(self, b):
+        return self.subAccumulate(b)
+
+    # mul(scalar)
+    def mul(self, scalar):
+        return Vector(self.x * scalar, self.y * scalar)
+
+    # a * scalar
+    def __mul__(self, scalar):
+        return self.mul(scalar)
+
+    # a *= scalar
+    def multAccumulate(self, scalar):
```

Comment:
Shouldn't this be named `mulAccumulate`? All of the multiplication methods in the rest of the class use `mul` as a prefix and not `mult`.